README.md

```diff
@@ -115,3 +115,23 @@ This code is distributed under the 2-clause BSD license.
 
 * Use the optional arguments to `python download-wgpu-native.py --help` to download a different version of the upstream wgpu-native binaries.
 * The file `wgpu/resources/wgpu_native-version` will be updated by the script to track which version we depend upon.
+
+### Example screenshot verification
+
+Include the comment `# test example: true` in an example to have pytest run it as part of the test suite.
+
+To support testing an example, ensure the following requirements are met:
+
+* The `WgpuCanvas` class is imported from the `wgpu.gui.auto` module.
+* The `canvas` instance is exposed as a global in the module.
+
+By default the test will simply verify that the example can be executed without an error, and that an
+image can be rendered using the canvas.
```

Comment:
And if a reference image is available it will be compared against that.

Reply:
That's what it says in the next couple of lines right?

Reply:
That's sort of what I mean by "enable", but I'll try and reword it

> If you additionally want to enable screenshot verification, you can generate the reference screenshot
like so: